wetmap/src/components/newModals/tripCreatorListModal/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { itineraries } from '../../../supabaseCalls/itinerarySupabaseCalls';
+import { updateDiveShop } from '../../../supabaseCalls/shopsSupabaseCalls';
+import { SelectedShopContext } from '../../contexts/selectedShopContext';
+import { UserProfileContext } from '../../contexts/userProfileContext';
+import { ModalContext } from '../../reusables/modal/context';
+import { ItineraryItem } from './types';
+import TripCreatorListView from './view';
+import { ModalHandleProps } from '../../reusables/modal/types';
+import DiveShopModal from '../shopModal/index';
+
+type TripCreatorListProps = Partial<ModalHandleProps>;
+
+export default function TripCreatorListModal(props: TripCreatorListProps) {
+  const { selectedShop } = useContext(SelectedShopContext);
```

Comment:
do we need to import this for some reason?

Reply:
Was just following the implementation in other index files. Used this as reference from the dive shop.

<img width="716" alt="image" src="https://github.com/user-attachments/assets/d79aed1a-91ec-4d8c-9abb-dff59a46fd6b" />

Reply:
It effectively does nothing though

Reply:
hmm ok let's get into the habit of removing code that we aren't using  no need to make it heavier with code that doesn't do anything